pkg/backup/client.go

```diff
@@ -183,12 +183,17 @@ func BuildBackupMeta(
 	files []*kvproto.File,
 	rawRanges []*kvproto.RawRange,
 	ddlJobs []*model.Job,
+	clusterVersion string,
+	brVersion string,
 ) (backupMeta kvproto.BackupMeta, err error) {
 	backupMeta.StartVersion = req.StartVersion
 	backupMeta.EndVersion = req.EndVersion
 	backupMeta.IsRawKv = req.IsRawKv
 	backupMeta.RawRanges = rawRanges
 	backupMeta.Files = files
+	backupMeta.ClusterId = req.ClusterId
+	backupMeta.ClusterVersion = clusterVersion
+	backupMeta.BrVersion = brVersion
```

Comment:
Please add a unit test to tests if these three fields are set correctly.

Reply:
addressed in https://github.com/pingcap/br/pull/734/commits/7b4365c010352b1c32d5639cb4ce366876692fbe